src/App.php

```diff
@@ -576,6 +576,21 @@ public function execute(Route $route, Request $request): self
                 }
             }
         } catch (\Throwable $e) {
+            if (empty(self::$errors['*'])) { // If no error handler is set then add a default one.
+                self::$errors['*'][] = [
```

Comment:
there is no longer `$errors['*']` please check the new implementation of errors.